templates/jupyterhub-instructions.md

```diff
@@ -315,44 +375,48 @@ Let's build and install the docker.
 
 Build the docker
 ----------------
-First copy the wheel into the docker dir..
+This builds a local docker.
 
-    $ cp dist/xpatterns_analytics_token_service-1.0-py2-none-any.whl docker/
-    $ cd docker
-    $ docker build -t xpatterns-token-service .
-    $ check by running "docker images"
+    $ cd ~/workspaces/xpatterns-analytics-token-service
+    $ bin/build-docker
 
-You should see xpatterns-analytics-token-service docker.
+Test by running
 
-This builds a local docker.
+    $ docker images
+
+You should see xpatterns-analytics-token-service docker.
 
-Later on, we will have Jenkins builds the docker that you are going to deploy, and
-it assigns a build number and puts it in the docker repo.
+Later on, we will have Jenkins build the docker that you are going to
+deploy, and it assigns a build number and puts it in the docker repo.
 
 We can run the service locally using:
 
-    $ docker/docker_run_script.sh
+    $ build/run-docker
+
+Test by running
 
+    $ docker ps
 
-Check that it is running with docker ps.
+This should show a running xpatterns-token-service.
 
 Test it by calling ping:
 
     $ curl localhost:16350/ping
 
 Should get pong.
 
-To build with jenkins, use
+To build with jenkins, go to
 
     http://jenkins.life.atigeo.com/job/xpatterns-token-service-docker/
 
 This does not trigger on checkin.
+Click on "Build With Parameters" to build it.
 
 Configure jupyterhub to use the token service
 ---------------------------------------------
-Go to julyterhub and add
+Go to julyterhub_config.py and add
```

Comment:
Should this really be "julyterhub_config.py"?  Or "jupyterhub_config.py" instead?  Looks like a typo to me.